wp-admin/js/widgets/media-image-widget.js

```diff
@@ -191,8 +191,11 @@
 			mediaFrame.state( 'replace-image' ).on( 'replace', updateCallback );
 			mediaFrame.on( 'close', function() {
 				mediaFrame.detach();
+				wp.media.model.Attachment.prototype.sync = defaultSync;
 			});
 
+			var defaultSync = wp.media.model.Attachment.prototype.sync;
+			wp.media.model.Attachment.prototype.sync = function() { return $.Deferred().rejectWith( this ).promise(); };
```

Comment:
@adamsilverstein correct me if I am wrong, but I don't believe that the changes here in `editMedia` are not necessary because the `image-details` state doesn't attempt to sync any of the changes back to the server, right?